Mono.Addins.Setup/Mono.Addins.Setup/SetupTool.cs

```diff
@@ -328,10 +328,7 @@ void ListInstalled (string[] args)
 			foreach (Addin addin in list) {
 				if (!showAll && IsHidden (addin))
 					continue;
-				Console.Write (" - " + addin.Name + " " + addin.Version);
-				if (showAll)
-					Console.Write (" (" + addin.AddinFile + ")");
-				Console.WriteLine ();
+				Console.WriteLine("{0} [{1}] {2}{3}", addin.Id, addin.Enabled ? "Enabled" : "Disabled", addin.Name, showAll ? $" ({addin.AddinFile})": string.Empty);
```

Comment:
```suggestion
				Console.WriteLine("[{0}] {1} {2}{3}", addin.Enabled ? "Enabled" : "Disabled", addin.Id, addin.Name, showAll ? $" ({addin.AddinFile})": string.Empty);
```

Reply:
Prefer writing the enabled/disabled status first, then the ID for at least some kind of symmetry.

Moved the space to the formatted string rather than the interpolated one, because it doesn't matter much when writing to console.